samples/serverReports/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Please use the Microsoft Source EULA for all code checked into this repo.  See the root directory for an example for the license.